R/plot_jags_tracedx.R

```diff
@@ -60,19 +60,23 @@ plot_jags_trace <- function(data,
         # Changing parameter name to reflect the input
         dplyr::mutate(Parameter = paste0("iso = ", j, ", parameter = ",
                                          .data$Parameter_sub, ", strat = ",
-                                         i),
-                      value = log(.data$value))
+                                         i))
       # Assigning attributes, which are needed to run ggs_density
       attributes(visualize_jags_plot) <- c(attributes(visualize_jags_plot),
                                            attributes_jags)
       # Creating density plot
       traceplot <- ggmcmc::ggs_traceplot(visualize_jags_plot) +
         ggplot2::theme_bw() +
-        ggplot2::labs(x = "log(value)") +
-        ggplot2::theme(legend.position = "bottom")
+        ggplot2::labs(x = "iterations", y = "log(value)") +
```

Comment:
The y axis tick marks are showing the parameter values on their original scale now, so I think we don't want `"log(value)"`, just `"value"`, or maybe `"parameter value"`)
```suggestion
        ggplot2::labs(x = "iterations", y = "parameter value") +
```